Misc/NEWS.d/next/Library/2025-05-14-08-13-08.gh-issue-133745.rjgJkH.rst

```diff
@@ -0,0 +1 @@
+In 3.13.3 we accidentally changed the signature of the asyncio ``create_task()`` family of methods and how it calls a custom task factory in a backwards incompatible way. Since some 3rd party libraries have already made changes to work around the issue that might break if we simply reverted the changes, we're instead changing things to be backwards compatible with 3.13.2 while still supporting those workarounds for 3.13.3. In particular, the special-casing of ``name`` and ``context`` is back (until 3.14) and consequently eager tasks may still find that their name hasn't been set before they execute their first yielding await.
```

Comment:
do we want to say ``create_task()` family of methods or just list out them all: `loop.create_task`, `TaskGroup.create_task` and `asyncio.create_task`

Reply:
I see no strong reason to spell them all out -- the 3.13.3 changes affect all methods named create_task and only those.